pyamlboot/pyamlboot.py

```diff
@@ -324,12 +333,21 @@ def tplCommand(self, subcode, command):
                                data_or_wLength = terminated_cmd)
 
     # tplStat
+    def tplStat(self, timeout=None):
+        return self.dev.ctrl_transfer(bmRequestType=0xc0,
+                                      bRequest=REQ_TPL_STAT,
+                                      wValue=0,
+                                      wIndex=0,
+                                      data_or_wLength=0x40,
+                                      timeout=timeout)
 
     def sendPassword(self, password):
         """UNTESTED: Send password"""
```

Comment:
I think you can remove this now you fixed it

Reply:
> I think you can remove this now you fixed it

Done